content/docs/contributing/release-process.md

```diff
@@ -468,16 +482,22 @@ page if a step is missing or if it is outdated.
 
     2. Generate `github-release-description.md` with the following command:
 
-        ```bash
-        # Must be run from the cert-manager folder.
-        export GITHUB_TOKEN=*your-token*
-        git fetch origin $BRANCH
-        export START_SHA="$(git rev-list --reverse --ancestry-path $(git merge-base $START_TAG $BRANCH)..$BRANCH | head -1)"
-        release-notes --debug --repo-path cert-manager \
-          --org cert-manager --repo cert-manager \
-          --required-author "jetstack-bot" \
-          --output github-release-description.md
-        ```
+       ```bash
+       # Must be run from the cert-manager folder.
+       export GITHUB_TOKEN=*your-token*
+       git fetch origin $BRANCH
+       export START_SHA="$(git rev-list --reverse --ancestry-path $(git merge-base $START_TAG $BRANCH)..$BRANCH | head -1)"
+       release-notes --debug --repo-path cert-manager \
+         --org cert-manager --repo cert-manager \
+         --required-author "jetstack-bot" \
+         --markdown-links=false \
+         --output github-release-description.md
+       ```
+
+       <div className="pageinfo pageinfo-info"><p>
+       The GitHub token **does not need any scope**. The token is required
+       only to avoid rate-limits imposed on anonymous API users.
+       </p></div>
```

Comment:
This section already exists, I think this is a copy-paste or git rebase error.

Reply:
Let's merge this PR and remove this section afterwards if that's OK

Reply:
@maelvls I mean that this section is just a copy: https://github.com/cert-manager/website/pull/1276/files/670e8b7816d93aaf080a8e5c3683205a9ab3778a#diff-fcf811a160f1a2424ea967f93210bbdbce5e726577411077634cb9a32848d2a7R502-R505

Why would we not just remove this from this PR?

Reply:
Ah, sorry, I didn't realize that I had made a mistake. I'll fix it in this PR.

Reply:
Done.